tests/compiler/test_opcodes.py

```diff
@@ -59,5 +59,8 @@ def test_get_opcodes(evm_version):
         assert "PUSH0" in ops
 
     if evm_version in ("cancun",):
-        assert "TLOAD" in ops
-        assert "TSTORE" in ops
+        for op in ("TLOAD", "TSTORE", "MCOPY"):
```

Comment:
A small thing: [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844), which will also be part of Cancun, introduces the opcode [`HASH_OPCODE_BYTE`](https://eips.ethereum.org/EIPS/eip-4844#opcode-to-get-versioned-hashes). For the sake of completeness, you should also add it here if we want to test it.

Reply:
we don't use it in any code generation routines, so i don't think it's necessary to add